source/_patterns/02-organisms/components/content-aside.mustache

```diff
@@ -1,4 +1,4 @@
-<aside class="content-aside" data-behaviour="ContentAside">
+<aside class="content-aside content-aside__sticky" data-behaviour="ContentAside">
```

Comment:
As above in line 27 in content-aside css this is not needed as the css can be placed in the breakpoint of 1000 and above of content-aside

Reply:
Done.